### PR TITLE
Keva: A simple key-value database

### PR DESCRIPTION
**Pull request by axblueblader**
image

### CODE REVIEW

File: keva/src/main/java/com/jinyframework/keva/Server.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+import java.io.InputStreamReader;
+import java.io.PrintWriter;
+import java.net.InetSocketAddress;
+import java.net.ServerSocket;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.Executors;
+
+@Slf4j
+public final class Server {
+    static String[] tokenized(String line) {
```
Comment (tuhuynh27) [inline keva/src/main/java/com/jinyframework/keva/Server.java:16]: Why, the name should be `tokenize`

File: keva/src/main/java/com/jinyframework/keva/storage/HashMapStorage.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import lombok.extern.slf4j.Slf4j;
+
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+
+@Slf4j
+public class HashMapStorage<K, V> {
+    private final ConcurrentMap<K, V> hashMap = new ConcurrentHashMap<>();
+
+    public byte put(K key, V val) {
```
Comment (tuhuynh27) [inline keva/src/main/java/com/jinyframework/keva/storage/HashMapStorage.java:12]: Why not `byte` => `bool`

Comment (axblueblader) [reply keva/src/main/java/com/jinyframework/keva/storage/HashMapStorage.java:12]: I wanted to return 1/0 like Redis instead of true/false string

**Review by tuhuynh27 — APPROVED** [review]
LGTM

**Pull request merged**